app/src/main/java/androidx/core/app/Person.java

```diff
@@ -424,7 +426,7 @@ static Person fromAndroidPerson(android.app.Person person) {
         static android.app.Person toAndroidPerson(Person person) {
             return new android.app.Person.Builder()
                 .setName(person.getName())
-                .setIcon((person.getIcon() != null) ? person.getIcon().toIcon() : null)
+                .setIcon((person.getIcon() != null) ? person.getIcon().toIcon(SocialApp.app) : null)
```

Comment:
Here is the real change, we have to call 

https://github.com/androidx/androidx/blob/4494d67f1ea1449c0206da02af1530bb94ab6cd1/core/core/src/main/java/androidx/core/graphics/drawable/IconCompat.java#L520-L534

on SDK 23+.